wp-content/themes/core/components/blocks/stats/Stats_Block_Controller.php

```diff
@@ -0,0 +1,178 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Templates\Components\blocks\stats;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use \Tribe\Project\Blocks\Types\Stats\Stats as Stats_Block;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use \Tribe\Project\Templates\Components\statistic\Controller as Statistic;
+use \Tribe\Project\Templates\Components\content_block\Controller as Content_Block;
+
+/**
+ * Class Hero
+ */
+class Stats_Block_Controller extends Abstract_Controller {
+	public const LAYOUT            = 'layout';
+	public const DISPLAY_DIVIDERS  = 'display_dividers';
+	public const CONTENT_ALIGN     = 'content_align';
+	public const TITLE             = 'title';
+	public const DESCRIPTION       = 'description';
+	public const CONTAINER_CLASSES = 'container_classes';
+	public const CONTENT_CLASSES   = 'content_classes';
+	public const CLASSES           = 'classes';
+	public const ATTRS             = 'attrs';
+	public const STATS             = 'stats';
+
+	public string $layout;
+	public string $content_align;
+	public string $display_dividers;
+	public string $title;
+	public string $description;
+	public array $container_classes;
+	public array $content_classes;
+	public array $classes;
+	public array $attrs;
+	public array $stats;
+
+	public function __construct( array $args = [] ) {
+		$args = wp_parse_args( $args, $this->defaults() );
```

Comment:
We have a method in Abstract_Controller that does all this now. We can do `$args = $this->parse_parts($args);`